### ISSUE TITLE
anime.js minification errors on 1.0.1

### ISSUE THREAD

**Issue by dlebedynskyi** [webpack-contrib/uglifyjs-webpack-plugin#162]
Hi, 
we noticed an issue with anime.js library start to crush on 1.0.1 while working fine on beta.2 

code in question after minification 
```
for (n in e)
                        y.hasOwnProperty(n) || "targets" === n || l.push({
                            name: n,
                            offset: y.offset,
                            tweens: function(e, t) {
                                var n = o(t);
                                if (v.arr(e)) {
                                    var r = e.length;
                                    2 !== r || v.obj(e[0]) ? v.fnc(t.duration) || (n.duration = t.duration / r) : e = {
                                        value: e
                                    }
                                }
                                return j(e).map(function(e, n) {
                                    return n = n ? 0 : t.delay,
                                    e = v.obj(e) && !v.pth(e) ? e : {
                                        value: e
                                    },
                                    v.und(e.delay) && (e.delay = n),
                                    e
                                }).map(function(e) {
                                    return s(e, n)
                                })
                            }(e[n], i)
                        });
```
Error in console... 

![image](https://user-images.githubusercontent.com/1182917/32735278-44876592-c862-11e7-8145-1020bc3e2c55.png)

Update: actually 1.0.0-rc and 1.0.0 both work as well.

Update2: issue seem to be in uglify-es  version itself .  > 3.1.3 is fine. 

I feel like i had older version in .lock files that satisfied deps for uglifyjs-webpack-plugin@1.0.1 but actually is not working well with it.  
Do you mind bumping version? 

_Feel free to close this if you feel like it_

**Comment by evilebottnawi** [comment]
@dlebedynskyi can you create minimum reproducible test repo?

**Comment by dlebedynskyi** [comment]
@evilebottnawi  I belive so far it was an issue with yarn.lock file itself. 
I have tried multiple options so far but from my investigation 

1. uglifyjs-webpack-plugin have deps as "uglify-es": "^3.1.3"
2. it seem that I had that dep resolve (or bad merge) to something  bigger then 3.1.3 but less then 3.1.9. From yarn.lock i see it should be 3.1.8. 
3. I got above with that. 

Once force update to 3.1.9 -all seem good. 
I'll try to find some time to make min repo
